randovania/data/json_data/prime1.json

```diff
@@ -803,33 +1563,148 @@
                             "name": "Dock to Subchamber Five",
                             "heal": false,
                             "coordinates": {
-                                "x": 33.74989414215088,
-                                "y": -23.983118057250977,
-                                "z": 22.500481825933093
+                                "x": 33.75,
+                                "y": -23.98,
+                                "z": 22.5
                             },
                             "node_type": "dock",
                             "dock_index": 0,
                             "connected_area_asset_id": 2003911832,
                             "connected_dock_index": 0,
-                            "dock_type": 0,
+                            "dock_type": 2,
                             "dock_weakness_index": 0,
                             "connections": {}
                         },
                         {
                             "name": "Dock to Subchamber Three",
                             "heal": false,
                             "coordinates": {
-                                "x": -33.74998950958252,
-                                "y": 0.7685813903808594,
-                                "z": 43.21452331542969
+                                "x": -33.75,
+                                "y": 0.77,
+                                "z": 43.21
                             },
                             "node_type": "dock",
                             "dock_index": 1,
                             "connected_area_asset_id": 2050677022,
                             "connected_dock_index": 0,
-                            "dock_type": 0,
-                            "dock_weakness_index": 0,
-                            "connections": {}
+                            "dock_type": 2,
+                            "dock_weakness_index": 0,
+                            "connections": {
+                                "Dock to Subchamber Five": {
+                                    "type": "and",
```

Comment:
this is being just a big "and": `Power Beam and Ice Beam and Wave Beam and Plasma Beam and Missile ≥ 50 and Flamethrower and Charge Beam and Knowledge (Beginner)`